main_routers/workshop_router.py

```
@@ -1123,293 +1424,300 @@ def _publish_workshop_item(steamworks, title, description, content_folder, previ
-        updated = [False]
-        error_code = [0]
-        update_event = threading.Event()
-        
-        def onSubmitItemUpdate(result):
-            nonlocal updated, error_code
-            # 直接从结构体读取字段而不是字典
-            error_code[0] = result.result
-            if result.result == 1:  # k_EResultOK
-                updated[0] = True
-                logger.info(f"物品更新提交成功，结果代码: {result.result}")
+            # 设置预览图片（如果提供）
```

Ruff (0.14.8)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)